ajo/PersonalizationPayloads.md

```diff
@@ -0,0 +1,396 @@
+## Personalization Payloads
+
+### Proposition Fetch
```

Comment:
Please improve this page, as a reader I have no idea what this document is trying to tell me or why this information matters.

Reply:
Agree this could use more context at the top. I would explain that it illustrates the Edge Network request/response payloads sent as part of the interaction flow in the provided samples. You could link to the Flow Diagram sections in the other readme docs for reference.